bin/trigger_azure_pipelines.py

```diff
@@ -74,7 +78,8 @@ def main() -> None:
         organization = os.environ["ADO_ORGANIZATION"]
         project = os.environ["ADO_PROJECT"]
         pipeline_id = os.environ["ADO_AZUREAUTH_LINUX_PIPELINE_ID"]
-
+        version = os.environ["VERSION"]
+        commitSHA = os.environ["commitSHA"]

```

Comment:
`$GITHUB_SHA` has the same information. You can use it directly. https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables